innopoints/views/variety.py

```diff
@@ -224,17 +248,24 @@ def edit_purchase_status(stock_change_id):
 
     stock_change = StockChange.query.get_or_404(stock_change_id)
     if stock_change.status != status:
+        variety = Variety.query.get(stock_change.variety_id)
+        product = variety.product
         if status == StockChangeStatus.rejected:
             db.session.delete(stock_change.transaction)
         elif stock_change.status == StockChangeStatus.rejected:
-            product = Variety.query.get(stock_change.variety_id).product
             new_transaction = Transaction(account=stock_change.account,
                                           change=product.price * stock_change.amount,
                                           stock_change_id=stock_change.id)
             stock_change.transaction = new_transaction
             db.session.add(new_transaction)
         stock_change.status = status
 
+        notify(stock_change.account_email, 'purchas_status_changed', {
```

Comment:
You have a typo in that string :) that's one of the reasons I don't like those hardcoded strings. I'll take care of it

Reply:
idk why, but it doesn't seem to cross my mind that an enum for that already exists 😅